### DISCUSSION TITLE
my deployment show difrent user name insted of mine. Hacked?

### DISCUSSION THREAD

**Discussion by itdieofficial** [vercel/community#1253] [Help] [answered]
### Summary

I've attached a screenshot of my deployments I do the last two deployments from my lap but it shows a different user name. "rfan-ansari-au28 " this git hub is not mine and my repository is private and I am the only one working on this project.
Do I need to be concerned about this?

<img width="1457" alt="Screenshot 2023-01-09 at 4 49 19 PM" src="https://user-images.githubusercontent.com/99642696/212058487-022f0184-ec9d-4c9a-92fe-41512f32db87.png">

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out @dieoVishnu. The git commit details are used to display the username, so misconfigured metadata is a common cause of this problem. Can you verify that you have [correctly configured metadata](https://vercel.com/guides/why-aren-t-commits-triggering-deployments-on-vercel#correctly-configured-metadata) by resetting the git config `user.name` and `user.email` on your local computer to match your GitHub account?

**Reply by itdieofficial** [reply]
I've reset the git config now it's working thank you

**Comment by akaushik759** [comment]
I also got the exact same username "irfan-ansari-au28"
very weird
